### DISCUSSION TITLE
useSendTransaction throws error about baseFee

### DISCUSSION THREAD

**Discussion by ahmadkhan345** [wevm/wagmi#2404] [Q&A]
I was using the useSendTransaction hook of wagmi in my app, it was working fine, but as soon as I entered the gasPrice parameter in the hook, I got an error, which stated something like that the baseFee was less than something

**Comment by tmm** [comment]
Want to share the error and a minimal reproduction? It's more likely someone will help you out if you provide more info.

**Reply by ahmadkhan345** [reply]
Ok thank you, will do that
